Twig/TwigYuiExtension.php

```diff
@@ -90,6 +90,14 @@ public function yuiConfigLoaderFunction($configObject = '')
             'modules' => [],
         ];
 
+        $combine = $this->configResolver->getParameter('yui.combine', 'ez_platformui');
+
+        if (true === $combine) {
```

Comment:
I really, really don't like this way to organize ifs... I know some people love it, but we don't use it in Platform (except a dozen ones in API tests), and I'd rather favour consistency.

I know it avoids mistakes, but on the other hand, it prevents you from writing something that won't work, so all in all, the benefit is really limited, and it's completely unnatural when you read it :-)